grace/simulator/store_features.py

```diff
@@ -0,0 +1,74 @@
+import click
+import torch
+
+from pathlib import Path
+from grace.base import GraphAttrs
+from grace.io import write_graph
+from grace.io.image_dataset import ImageGraphDataset
+from grace.models.feature_extractor import FeatureExtractor
+
+
+# Define a click command to input the file name directly:
+@click.command(name="Graph Storage")
+@click.option(
+    "--data_path",
+    type=click.Path(exists=True),
+    help="Path to images and grace annotations",
+)
+@click.option(
+    "--extractor_fn",
+    type=click.Path(exists=True),
+    help="Path to feature extractor model",
+)
+@click.option(
+    "--extractor_fn",
```

Comment:
why does this option have the same name as the one above? (line 19)

Reply:
Typo - well spotted! :) Fix incoming!